site/content/en/docs/installation/_index.md

```diff
@@ -243,22 +243,23 @@ spec:
 
 The currently supported features are:
 
-| Feature | Default | Stage | Since | Until |
```

Comment:
Could you please reformat the table in a dedicated PR? So that it is clear what is changed.

Reply:
Done

Reply:
Done on https://github.com/kubernetes-sigs/kueue/pull/3048.